ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-evpn.yml

```diff
@@ -60,6 +60,8 @@ router_bgp:
       missing_policy:
         direction_out:
           action: deny-in-out
+      route_map_in: RM1
+      route_map_out: RM2
```

Comment:
You are adding the test case at wrong place, it needs to be added under `address_family_evpn.neighbors`

Reply:
Fixed.